  INFO root [0x00003ab4] [2018-07-31 16:16:22,284] - 
  INFO root [0x00003ab4] [2018-07-31 16:16:22,285] - -----------------------------------------------------------------------------------------


  INFO root [0x00003ab4] [2018-07-31 16:16:22,288] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003ab4] [2018-07-31 16:16:22,289] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00003ab4] [2018-07-31 16:16:22,289] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003ab4] [2018-07-31 16:16:22,297] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:20015 )
  INFO root [0x00003ab4] [2018-07-31 16:16:22,304] - NetworkInterface::initialize(EXTERNAL): address 10.139.66.192:20015, SOMAXCONN=128.
  INFO root [0x00003ab4] [2018-07-31 16:16:22,319] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:42712 )
  INFO root [0x00003ab4] [2018-07-31 16:16:22,319] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:42712, SOMAXCONN=128.
  INFO root [0x00003ab4] [2018-07-31 16:16:22,320] - server-configs:
  INFO root [0x00003ab4] [2018-07-31 16:16:22,320] - 	gameUpdateHertz : 10
  INFO root [0x00003ab4] [2018-07-31 16:16:22,320] - 	entryScriptFile : kbemain
  INFO root [0x00003ab4] [2018-07-31 16:16:22,321] - 	internalAddr : 10.139.66.192:42712
  INFO root [0x00003ab4] [2018-07-31 16:16:22,321] - 	externalAddr : 10.139.66.192:20015
  INFO root [0x00003ab4] [2018-07-31 16:16:22,321] - 	componentID : 7001
  INFO root [0x00003ab4] [2018-07-31 16:16:22,550] - Components::findLogger: found logger, addr:10.139.66.192:42675
  INFO root [0x00003ab4] [2018-07-31 16:16:22,551] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00003ab4] [2018-07-31 16:16:22,551] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003ab4] [2018-07-31 16:16:22,569] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 16:27:53 Apr 30 2018. AppID: 7001. UID: 10103. PID: 11228 ----
  INFO root [0x00003ab4] [2018-07-31 16:16:22,570] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00003ab4] [2018-07-31 16:16:22,571] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003ab4] [2018-07-31 16:16:22,572] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,735] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,735] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,736] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,744] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,745] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,745] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,746] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003ab4] [2018-07-31 16:16:22,816] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00003ab4] [2018-07-31 16:16:22,892] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:27) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00003ab4] [2018-07-31 16:16:22,893] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003ab4] [2018-07-31 16:16:23,169] - loaded entity-script:Account(1).
 DEBUG root [0x00003ab4] [2018-07-31 16:16:23,217] - loaded entity-script:Avatar(2).
 DEBUG root [0x00003ab4] [2018-07-31 16:16:23,823] - loaded entity-script:Equipment(3).
 DEBUG root [0x00003ab4] [2018-07-31 16:16:23,875] - loaded entity-script:Chat(4).
  INFO root [0x00003ab4] [2018-07-31 16:16:23,876] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003ab4] [2018-07-31 16:16:23,877] - TelnetServer server is running on port 40000
  INFO root [0x00003ab4] [2018-07-31 16:16:24,519] - ---- baseapp is running ----
 DEBUG root [0x00003ab4] [2018-07-31 16:16:24,519] - Components::process(): Request for the process of identity...
  WARN root [0x00003ab4] [2018-07-31 16:16:30,778] - Forwarding logs to logger[10.139.66.192:42675/0/0/0]...
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,422] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00003ab4] [2018-07-31 16:20:00,423] - The following logs sent to logger failed:
  WARN root [0x00003ab4] [2018-07-31 16:20:00,424] - ==> [2018-07-31 16:20:00 421] TCPPacketReceiver::processPendingEvents(10.139.66.192:42675): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,424] - ==> [2018-07-31 16:20:00 423] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00003ab4] [2018-07-31 16:20:00,570] - TCPPacketReceiver::processPendingEvents(10.139.66.192:42686): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,571] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00003ab4] [2018-07-31 16:20:00,579] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00003ab4] [2018-07-31 16:20:00,585] - onBaseAppShutDown: state=0
  INFO root [0x00003ab4] [2018-07-31 16:20:00,585] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00003ab4] [2018-07-31 16:20:00,585] - onBaseAppShutDown: state=1
  INFO root [0x00003ab4] [2018-07-31 16:20:00,585] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00003ab4] [2018-07-31 16:20:00,586] - onBaseAppShutDown: state=2
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,611] - Equipment::onCellWriteToDBCompleted(1): not found dbmgr!
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,614] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,622] - Account::onCellWriteToDBCompleted(3): not found dbmgr!
 ERROR root [0x00003ab4] [2018-07-31 16:20:00,624] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,630] - Account::~Account(): 3
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,630] - Equipment::~Equipment(): 1
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,630] - Chat::~Chat(): 2
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,632] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,632] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,633] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,633] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,633] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,633] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,633] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,634] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00003ab4] [2018-07-31 16:20:00,652] - Script::uninstall(): is successfully!
 DEBUG root [0x00003ab4] [2018-07-31 16:20:00,652] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00003ab4] [2018-07-31 16:20:00,954] - ThreadPool::destroy(): waiting for thread(2)[0x227eae0,0x227ee20,], try=1
  INFO root [0x00001ec0] [2018-07-31 16:20:03,418] - 
  INFO root [0x00001ec0] [2018-07-31 16:20:03,419] - -----------------------------------------------------------------------------------------


  INFO root [0x00001ec0] [2018-07-31 16:20:03,420] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00001ec0] [2018-07-31 16:20:03,420] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00001ec0] [2018-07-31 16:20:03,420] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00001ec0] [2018-07-31 16:20:03,428] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:20015 )
  INFO root [0x00001ec0] [2018-07-31 16:20:03,428] - NetworkInterface::initialize(EXTERNAL): address 10.139.66.192:20015, SOMAXCONN=128.
  INFO root [0x00001ec0] [2018-07-31 16:20:03,442] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:43417 )
  INFO root [0x00001ec0] [2018-07-31 16:20:03,443] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:43417, SOMAXCONN=128.
  INFO root [0x00001ec0] [2018-07-31 16:20:03,443] - server-configs:
  INFO root [0x00001ec0] [2018-07-31 16:20:03,443] - 	gameUpdateHertz : 10
  INFO root [0x00001ec0] [2018-07-31 16:20:03,443] - 	entryScriptFile : kbemain
  INFO root [0x00001ec0] [2018-07-31 16:20:03,443] - 	internalAddr : 10.139.66.192:43417
  INFO root [0x00001ec0] [2018-07-31 16:20:03,444] - 	externalAddr : 10.139.66.192:20015
  INFO root [0x00001ec0] [2018-07-31 16:20:03,444] - 	componentID : 7001
  INFO root [0x00001ec0] [2018-07-31 16:20:03,751] - Components::findLogger: found logger, addr:10.139.66.192:43411
  INFO root [0x00001ec0] [2018-07-31 16:20:03,751] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00001ec0] [2018-07-31 16:20:03,751] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001ec0] [2018-07-31 16:20:03,770] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 16:27:53 Apr 30 2018. AppID: 7001. UID: 10103. PID: 9052 ----
  INFO root [0x00001ec0] [2018-07-31 16:20:03,770] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00001ec0] [2018-07-31 16:20:03,771] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001ec0] [2018-07-31 16:20:03,772] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,937] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,938] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,939] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,943] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,944] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,944] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001ec0] [2018-07-31 16:20:03,945] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001ec0] [2018-07-31 16:20:04,054] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001ec0] [2018-07-31 16:20:04,133] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:27) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00001ec0] [2018-07-31 16:20:04,133] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001ec0] [2018-07-31 16:20:04,193] - loaded entity-script:Account(1).
 DEBUG root [0x00001ec0] [2018-07-31 16:20:04,196] - loaded entity-script:Avatar(2).
 DEBUG root [0x00001ec0] [2018-07-31 16:20:04,213] - loaded entity-script:Equipment(3).
 DEBUG root [0x00001ec0] [2018-07-31 16:20:04,218] - loaded entity-script:Chat(4).
  INFO root [0x00001ec0] [2018-07-31 16:20:04,218] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001ec0] [2018-07-31 16:20:04,219] - TelnetServer server is running on port 40000
  INFO root [0x00001ec0] [2018-07-31 16:20:04,929] - ---- baseapp is running ----
 DEBUG root [0x00001ec0] [2018-07-31 16:20:04,929] - Components::process(): Request for the process of identity...
  WARN root [0x00001ec0] [2018-07-31 16:20:11,229] - Forwarding logs to logger[10.139.66.192:43411/0/0/0]...
 ERROR root [0x00001ec0] [2018-07-31 16:34:31,970] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001ec0] [2018-07-31 16:34:31,973] - The following logs sent to logger failed:
  WARN root [0x00001ec0] [2018-07-31 16:34:31,974] - ==> [2018-07-31 16:34:31 968] TCPPacketReceiver::processPendingEvents(10.139.66.192:43411): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001ec0] [2018-07-31 16:34:31,974] - ==> [2018-07-31 16:34:31 971] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001ec0] [2018-07-31 16:34:32,149] - TCPPacketReceiver::processPendingEvents(10.139.66.192:43413): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001ec0] [2018-07-31 16:34:32,156] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001ec0] [2018-07-31 16:34:32,158] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001ec0] [2018-07-31 16:34:32,160] - onBaseAppShutDown: state=0
  INFO root [0x00001ec0] [2018-07-31 16:34:32,160] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001ec0] [2018-07-31 16:34:32,161] - onBaseAppShutDown: state=1
  INFO root [0x00001ec0] [2018-07-31 16:34:32,162] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001ec0] [2018-07-31 16:34:32,162] - onBaseAppShutDown: state=2
 ERROR root [0x00001ec0] [2018-07-31 16:34:32,195] - Equipment::onCellWriteToDBCompleted(2001): not found dbmgr!
 ERROR root [0x00001ec0] [2018-07-31 16:34:32,213] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001ec0] [2018-07-31 16:34:32,217] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00001ec0] [2018-07-31 16:34:32,219] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,224] - Account::~Account(): 2003
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,225] - Equipment::~Equipment(): 2001
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,225] - Chat::~Chat(): 2002
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,228] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,229] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,229] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,230] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,230] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,230] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,230] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,230] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001ec0] [2018-07-31 16:34:32,262] - Script::uninstall(): is successfully!
 DEBUG root [0x00001ec0] [2018-07-31 16:34:32,262] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00001ec0] [2018-07-31 16:34:32,568] - ThreadPool::destroy(): waiting for thread(2)[0x2426cb8,0x2426ab0,], try=1
  INFO root [0x0000342c] [2018-07-31 16:34:35,454] - 
  INFO root [0x0000342c] [2018-07-31 16:34:35,456] - -----------------------------------------------------------------------------------------


  INFO root [0x0000342c] [2018-07-31 16:34:35,458] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x0000342c] [2018-07-31 16:34:35,459] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x0000342c] [2018-07-31 16:34:35,461] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x0000342c] [2018-07-31 16:34:35,511] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:20015 )
  INFO root [0x0000342c] [2018-07-31 16:34:35,512] - NetworkInterface::initialize(EXTERNAL): address 10.139.66.192:20015, SOMAXCONN=128.
  INFO root [0x0000342c] [2018-07-31 16:34:35,529] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:46224 )
  INFO root [0x0000342c] [2018-07-31 16:34:35,532] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:46224, SOMAXCONN=128.
  INFO root [0x0000342c] [2018-07-31 16:34:35,533] - server-configs:
  INFO root [0x0000342c] [2018-07-31 16:34:35,533] - 	gameUpdateHertz : 10
  INFO root [0x0000342c] [2018-07-31 16:34:35,533] - 	entryScriptFile : kbemain
  INFO root [0x0000342c] [2018-07-31 16:34:35,533] - 	internalAddr : 10.139.66.192:46224
  INFO root [0x0000342c] [2018-07-31 16:34:35,533] - 	externalAddr : 10.139.66.192:20015
  INFO root [0x0000342c] [2018-07-31 16:34:35,534] - 	componentID : 7001
  INFO root [0x0000342c] [2018-07-31 16:34:36,314] - Components::findLogger: found logger, addr:10.139.66.192:46218
  INFO root [0x0000342c] [2018-07-31 16:34:36,315] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x0000342c] [2018-07-31 16:34:36,315] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000342c] [2018-07-31 16:34:36,336] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 16:27:53 Apr 30 2018. AppID: 7001. UID: 10103. PID: 6044 ----
  INFO root [0x0000342c] [2018-07-31 16:34:36,347] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x0000342c] [2018-07-31 16:34:36,347] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000342c] [2018-07-31 16:34:36,348] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,514] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,515] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,516] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,524] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,524] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,525] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,525] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,625] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000342c] [2018-07-31 16:34:36,725] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:27) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x0000342c] [2018-07-31 16:34:36,726] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,798] - loaded entity-script:Account(1).
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,808] - loaded entity-script:Avatar(2).
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,831] - loaded entity-script:Equipment(3).
 DEBUG root [0x0000342c] [2018-07-31 16:34:36,844] - loaded entity-script:Chat(4).
  INFO root [0x0000342c] [2018-07-31 16:34:36,844] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000342c] [2018-07-31 16:34:36,848] - TelnetServer server is running on port 40000
  INFO root [0x0000342c] [2018-07-31 16:34:38,200] - ---- baseapp is running ----
 DEBUG root [0x0000342c] [2018-07-31 16:34:38,208] - Components::process(): Request for the process of identity...
  INFO root [0x00003af8] [2018-07-31 16:41:00,170] - 
  INFO root [0x00003af8] [2018-07-31 16:41:00,170] - -----------------------------------------------------------------------------------------


  INFO root [0x00003af8] [2018-07-31 16:41:00,171] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00003af8] [2018-07-31 16:41:00,172] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00003af8] [2018-07-31 16:41:00,172] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00003af8] [2018-07-31 16:41:00,177] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:20015 )
  INFO root [0x00003af8] [2018-07-31 16:41:00,178] - NetworkInterface::initialize(EXTERNAL): address 10.139.66.192:20015, SOMAXCONN=128.
  INFO root [0x00003af8] [2018-07-31 16:41:00,191] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:47464 )
  INFO root [0x00003af8] [2018-07-31 16:41:00,191] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:47464, SOMAXCONN=128.
  INFO root [0x00003af8] [2018-07-31 16:41:00,191] - server-configs:
  INFO root [0x00003af8] [2018-07-31 16:41:00,191] - 	gameUpdateHertz : 10
  INFO root [0x00003af8] [2018-07-31 16:41:00,191] - 	entryScriptFile : kbemain
  INFO root [0x00003af8] [2018-07-31 16:41:00,191] - 	internalAddr : 10.139.66.192:47464
  INFO root [0x00003af8] [2018-07-31 16:41:00,192] - 	externalAddr : 10.139.66.192:20015
  INFO root [0x00003af8] [2018-07-31 16:41:00,192] - 	componentID : 7001
  INFO root [0x00003af8] [2018-07-31 16:41:00,600] - Components::findLogger: found logger, addr:10.139.66.192:47458
  INFO root [0x00003af8] [2018-07-31 16:41:00,600] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00003af8] [2018-07-31 16:41:00,600] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003af8] [2018-07-31 16:41:00,620] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 16:27:53 Apr 30 2018. AppID: 7001. UID: 10103. PID: 17264 ----
  INFO root [0x00003af8] [2018-07-31 16:41:00,621] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00003af8] [2018-07-31 16:41:00,621] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003af8] [2018-07-31 16:41:00,621] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,766] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,766] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,767] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,771] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,771] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,771] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,772] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,829] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00003af8] [2018-07-31 16:41:00,893] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:27) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00003af8] [2018-07-31 16:41:00,893] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,937] - loaded entity-script:Account(1).
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,939] - loaded entity-script:Avatar(2).
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,953] - loaded entity-script:Equipment(3).
 DEBUG root [0x00003af8] [2018-07-31 16:41:00,956] - loaded entity-script:Chat(4).
  INFO root [0x00003af8] [2018-07-31 16:41:00,956] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003af8] [2018-07-31 16:41:00,957] - TelnetServer server is running on port 40000
  INFO root [0x00003af8] [2018-07-31 16:41:01,587] - ---- baseapp is running ----
 DEBUG root [0x00003af8] [2018-07-31 16:41:01,588] - Components::process(): Request for the process of identity...
  WARN root [0x00003af8] [2018-07-31 16:41:07,947] - Forwarding logs to logger[10.139.66.192:47458/0/0/0]...
 ERROR root [0x00003af8] [2018-07-31 17:18:46,158] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00003af8] [2018-07-31 17:18:46,166] - The following logs sent to logger failed:
  WARN root [0x00003af8] [2018-07-31 17:18:46,166] - ==> [2018-07-31 17:18:46 156] TCPPacketReceiver::processPendingEvents(10.139.66.192:47458): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003af8] [2018-07-31 17:18:46,166] - ==> [2018-07-31 17:18:46 159] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00003af8] [2018-07-31 17:18:46,320] - TCPPacketReceiver::processPendingEvents(10.139.66.192:47459): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003af8] [2018-07-31 17:18:46,323] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00003af8] [2018-07-31 17:18:46,325] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00003af8] [2018-07-31 17:18:46,326] - onBaseAppShutDown: state=0
  INFO root [0x00003af8] [2018-07-31 17:18:46,326] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00003af8] [2018-07-31 17:18:46,327] - onBaseAppShutDown: state=1
  INFO root [0x00003af8] [2018-07-31 17:18:46,327] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00003af8] [2018-07-31 17:18:46,327] - onBaseAppShutDown: state=2
 ERROR root [0x00003af8] [2018-07-31 17:18:46,361] - Equipment::onCellWriteToDBCompleted(2001): not found dbmgr!
 ERROR root [0x00003af8] [2018-07-31 17:18:46,363] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00003af8] [2018-07-31 17:18:46,364] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00003af8] [2018-07-31 17:18:46,364] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,364] - Account::~Account(): 2003
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,365] - Equipment::~Equipment(): 2001
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,366] - Chat::~Chat(): 2002
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,368] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,369] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,370] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,370] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,370] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,371] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,371] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,371] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00003af8] [2018-07-31 17:18:46,397] - Script::uninstall(): is successfully!
 DEBUG root [0x00003af8] [2018-07-31 17:18:46,398] - ThreadPool::destroy(): starting size 2.
  WARN root [0x00003af8] [2018-07-31 17:18:46,698] - ThreadPool::destroy(): waiting for thread(2)[0x250aab0,0x250abe8,], try=1
  INFO root [0x000042dc] [2018-07-31 17:18:51,338] - 
  INFO root [0x000042dc] [2018-07-31 17:18:51,342] - -----------------------------------------------------------------------------------------


  INFO root [0x000042dc] [2018-07-31 17:18:51,344] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x000042dc] [2018-07-31 17:18:51,345] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x000042dc] [2018-07-31 17:18:51,345] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x000042dc] [2018-07-31 17:18:51,359] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:20015 )
  INFO root [0x000042dc] [2018-07-31 17:18:51,359] - NetworkInterface::initialize(EXTERNAL): address 10.139.66.192:20015, SOMAXCONN=128.
  INFO root [0x000042dc] [2018-07-31 17:18:51,375] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:35577 )
  INFO root [0x000042dc] [2018-07-31 17:18:51,376] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:35577, SOMAXCONN=128.
  INFO root [0x000042dc] [2018-07-31 17:18:51,376] - server-configs:
  INFO root [0x000042dc] [2018-07-31 17:18:51,376] - 	gameUpdateHertz : 10
  INFO root [0x000042dc] [2018-07-31 17:18:51,376] - 	entryScriptFile : kbemain
  INFO root [0x000042dc] [2018-07-31 17:18:51,376] - 	internalAddr : 10.139.66.192:35577
  INFO root [0x000042dc] [2018-07-31 17:18:51,376] - 	externalAddr : 10.139.66.192:20015
  INFO root [0x000042dc] [2018-07-31 17:18:51,377] - 	componentID : 7001
  INFO root [0x000042dc] [2018-07-31 17:18:51,652] - Components::findLogger: found logger, addr:10.139.66.192:35562
  INFO root [0x000042dc] [2018-07-31 17:18:51,653] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x000042dc] [2018-07-31 17:18:51,654] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000042dc] [2018-07-31 17:18:51,682] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 16:27:53 Apr 30 2018. AppID: 7001. UID: 10103. PID: 9348 ----
  INFO root [0x000042dc] [2018-07-31 17:18:51,682] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x000042dc] [2018-07-31 17:18:51,684] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000042dc] [2018-07-31 17:18:51,685] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,861] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,862] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,862] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,868] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,869] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,869] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,870] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000042dc] [2018-07-31 17:18:51,994] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000042dc] [2018-07-31 17:18:52,111] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:27) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x000042dc] [2018-07-31 17:18:52,112] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000042dc] [2018-07-31 17:18:52,190] - loaded entity-script:Account(1).
 DEBUG root [0x000042dc] [2018-07-31 17:18:52,193] - loaded entity-script:Avatar(2).
 DEBUG root [0x000042dc] [2018-07-31 17:18:52,232] - loaded entity-script:Equipment(3).
 DEBUG root [0x000042dc] [2018-07-31 17:18:52,236] - loaded entity-script:Chat(4).
  INFO root [0x000042dc] [2018-07-31 17:18:52,237] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000042dc] [2018-07-31 17:18:52,238] - TelnetServer server is running on port 40000
  INFO root [0x000042dc] [2018-07-31 17:18:53,057] - ---- baseapp is running ----
 DEBUG root [0x000042dc] [2018-07-31 17:18:53,058] - Components::process(): Request for the process of identity...
  WARN root [0x000042dc] [2018-07-31 17:18:59,739] - Forwarding logs to logger[10.139.66.192:35562/0/0/0]...
 ERROR root [0x000042dc] [2018-07-31 17:35:59,275] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000042dc] [2018-07-31 17:35:59,278] - The following logs sent to logger failed:
  WARN root [0x000042dc] [2018-07-31 17:35:59,279] - ==> [2018-07-31 17:35:59 270] TCPPacketReceiver::processPendingEvents(10.139.66.192:35562): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000042dc] [2018-07-31 17:35:59,279] - ==> [2018-07-31 17:35:59 275] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000042dc] [2018-07-31 17:35:59,412] - TCPPacketReceiver::processPendingEvents(10.139.66.192:35568): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000042dc] [2018-07-31 17:35:59,414] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000042dc] [2018-07-31 17:35:59,419] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000042dc] [2018-07-31 17:35:59,420] - onBaseAppShutDown: state=0
  INFO root [0x000042dc] [2018-07-31 17:35:59,421] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000042dc] [2018-07-31 17:35:59,421] - onBaseAppShutDown: state=1
  INFO root [0x000042dc] [2018-07-31 17:35:59,421] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000042dc] [2018-07-31 17:35:59,421] - onBaseAppShutDown: state=2
 ERROR root [0x000042dc] [2018-07-31 17:35:59,440] - Equipment::onCellWriteToDBCompleted(2001): not found dbmgr!
 ERROR root [0x000042dc] [2018-07-31 17:35:59,441] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x000042dc] [2018-07-31 17:35:59,441] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x000042dc] [2018-07-31 17:35:59,446] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,447] - Account::~Account(): 2003
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,447] - Avatar::~Avatar(): 2004
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,447] - Equipment::~Equipment(): 2001
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,448] - Chat::~Chat(): 2002
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,449] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,449] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,449] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,449] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,449] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,450] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,450] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,450] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000042dc] [2018-07-31 17:35:59,468] - Script::uninstall(): is successfully!
 DEBUG root [0x000042dc] [2018-07-31 17:35:59,468] - ThreadPool::destroy(): starting size 2.
  WARN root [0x000042dc] [2018-07-31 17:35:59,768] - ThreadPool::destroy(): waiting for thread(2)[0x2442c60,0x24429f0,], try=1
  INFO root [0x00001924] [2018-07-31 17:36:02,236] - 
  INFO root [0x00001924] [2018-07-31 17:36:02,238] - -----------------------------------------------------------------------------------------


  INFO root [0x00001924] [2018-07-31 17:36:02,242] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [0x00001924] [2018-07-31 17:36:02,244] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/assets_Battle of Gods/;D:/kbengine/assets_Battle of Gods/scripts/;D:/kbengine/assets_Battle of Gods/res/
  INFO root [0x00001924] [2018-07-31 17:36:02,244] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [0x00001924] [2018-07-31 17:36:02,262] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:20015 )
  INFO root [0x00001924] [2018-07-31 17:36:02,262] - NetworkInterface::initialize(EXTERNAL): address 10.139.66.192:20015, SOMAXCONN=128.
  INFO root [0x00001924] [2018-07-31 17:36:02,290] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 10.139.66.192 ( 10.139.66.192:2557 )
  INFO root [0x00001924] [2018-07-31 17:36:02,291] - NetworkInterface::initialize(INTERNAL): address 10.139.66.192:2557, SOMAXCONN=128.
  INFO root [0x00001924] [2018-07-31 17:36:02,291] - server-configs:
  INFO root [0x00001924] [2018-07-31 17:36:02,291] - 	gameUpdateHertz : 10
  INFO root [0x00001924] [2018-07-31 17:36:02,291] - 	entryScriptFile : kbemain
  INFO root [0x00001924] [2018-07-31 17:36:02,292] - 	internalAddr : 10.139.66.192:2557
  INFO root [0x00001924] [2018-07-31 17:36:02,292] - 	externalAddr : 10.139.66.192:20015
  INFO root [0x00001924] [2018-07-31 17:36:02,292] - 	componentID : 7001
  INFO root [0x00001924] [2018-07-31 17:36:02,976] - Components::findLogger: found logger, addr:10.139.66.192:2551
  INFO root [0x00001924] [2018-07-31 17:36:02,977] - Components::addComponent: reset orderLog, uid=10103!
  INFO root [0x00001924] [2018-07-31 17:36:02,977] - Components::addComponent[logger], uid=10103, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001924] [2018-07-31 17:36:02,996] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 16:27:53 Apr 30 2018. AppID: 7001. UID: 10103. PID: 13544 ----
  INFO root [0x00001924] [2018-07-31 17:36:02,996] - Server : Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz with 2097151kB RAM
  INFO root [0x00001924] [2018-07-31 17:36:02,997] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001924] [2018-07-31 17:36:02,998] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001924] [2018-07-31 17:36:03,154] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001924] [2018-07-31 17:36:03,155] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001924] [2018-07-31 17:36:03,155] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001924] [2018-07-31 17:36:03,159] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001924] [2018-07-31 17:36:03,159] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001924] [2018-07-31 17:36:03,160] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001924] [2018-07-31 17:36:03,160] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001924] [2018-07-31 17:36:03,223] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001924] [2018-07-31 17:36:03,282] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 30 2018, 16:27:27) [MSC v.1900 32 bit (Intel)])!
  INFO root [0x00001924] [2018-07-31 17:36:03,282] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001924] [2018-07-31 17:36:03,343] - loaded entity-script:Account(1).
 DEBUG root [0x00001924] [2018-07-31 17:36:03,345] - loaded entity-script:Avatar(2).
 DEBUG root [0x00001924] [2018-07-31 17:36:03,364] - loaded entity-script:Equipment(3).
 DEBUG root [0x00001924] [2018-07-31 17:36:03,369] - loaded entity-script:Chat(4).
  INFO root [0x00001924] [2018-07-31 17:36:03,369] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001924] [2018-07-31 17:36:03,370] - TelnetServer server is running on port 40000
  INFO root [0x00001924] [2018-07-31 17:36:04,076] - ---- baseapp is running ----
 DEBUG root [0x00001924] [2018-07-31 17:36:04,076] - Components::process(): Request for the process of identity...
  WARN root [0x00001924] [2018-07-31 17:36:10,458] - Forwarding logs to logger[10.139.66.192:2551/0/0/0]...
 ERROR root [0x00001924] [2018-07-31 17:40:50,309] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001924] [2018-07-31 17:40:50,319] - The following logs sent to logger failed:
  WARN root [0x00001924] [2018-07-31 17:40:50,319] - ==> [2018-07-31 17:40:50 317] TCPPacketReceiver::processPendingEvents(10.139.66.192:2551): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001924] [2018-07-31 17:40:50,323] - ==> [2018-07-31 17:40:50 319] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001924] [2018-07-31 17:40:50,559] - TCPPacketReceiver::processPendingEvents(10.139.66.192:2552): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001924] [2018-07-31 17:40:50,559] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001924] [2018-07-31 17:40:50,563] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001924] [2018-07-31 17:40:50,563] - onBaseAppShutDown: state=0
  INFO root [0x00001924] [2018-07-31 17:40:50,563] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001924] [2018-07-31 17:40:50,563] - onBaseAppShutDown: state=1
  INFO root [0x00001924] [2018-07-31 17:40:50,563] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001924] [2018-07-31 17:40:50,563] - onBaseAppShutDown: state=2
 ERROR root [0x00001924] [2018-07-31 17:40:50,589] - Equipment::onCellWriteToDBCompleted(2001): not found dbmgr!
 ERROR root [0x00001924] [2018-07-31 17:40:50,589] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001924] [2018-07-31 17:40:50,589] - Account::onCellWriteToDBCompleted(2003): not found dbmgr!
 ERROR root [0x00001924] [2018-07-31 17:40:50,599] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001924] [2018-07-31 17:40:50,599] - Account::~Account(): 2003
 DEBUG root [0x00001924] [2018-07-31 17:40:50,599] - Equipment::~Equipment(): 2001
 DEBUG root [0x00001924] [2018-07-31 17:40:50,599] - Chat::~Chat(): 2002
 DEBUG root [0x00001924] [2018-07-31 17:40:50,599] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001924] [2018-07-31 17:40:50,609] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001924] [2018-07-31 17:40:50,619] - Script::uninstall(): is successfully!
 DEBUG root [0x00001924] [2018-07-31 17:40:50,619] - ThreadPool::destroy(): starting size 2.
